taittirIyam/sUtram/baudhAyanaH/brahma-karma-samuchchayaH/3_sankIrNakANDam/22_mUlanaxatrajananashAntividhiH.md

```diff
@@ -2,78 +2,118 @@
+भरेष्विन्द्रगꣳ सुहवागाꣳ हवामहे ।  
+अꣳ होमुचꣳ सुकृतं दैव्यञ्जनम् ।  
+अग्निं मित्रं वरुणꣳ सातये भगम् ।  
+द्यावापृथिवी मरुतः स्वस्तये ॥ ७ ॥  
+महि क्षेत्रं पुरुश्चन्द्रं विविद्वान् ।  
+आदित्सखिभ्यश्च स्थर समैरत् ।  
+इन्द्रो नृभिरजनुद्दीघानः साकम् ।  
+सूर्य मुषसं गातुमग्निम् ॥ ८ ॥  
+ऊरुन्नो लोकमनुनेषि विद्वान् ।  
+सुवर्वज्ज्योतिरभयꣳ स्वस्ति ।  
+ऋण्वा त इन्द्र स्थविरस्य बाहू ।  
+उपस्थे याम शरणा बृहन्ता ॥ ९ ॥  
+आ नो विश्वाभिरूतिभिः सजोषाः ।  
+ब्रह्म जुषाण्णो हर्यश्व याहि ।  
+वरीवृजत्स्थविरेमिः सुशिप्र ॥  
+अस्मे दधद्वृषणꣳ शुष्ममिन्द्र ॥ १० ॥  
+इन्द्राय गाव आशिरम् ।  
+दुदुह्रे वज्रिणे मधु।  
+यत्सीमुपह्णरेऽविदत् ॥ ११ ॥  
+तास्ते वनिन्धेनवो जोजयुर्नः।  
+गर्भस्तयो नियुतो विश्वाराः ।  
+अहरहर्भूय इज्जोगुवानाः।  
+पूर्णा इन्द्र क्षुमतो भोजनस्य ॥ १२ ॥  
+इमां ते धियं प्रभरे महो महीम् ।  
+अस्य स्तोत्रे धिषणा यत्त आनजे।  
+तमुत्सवे च प्रसवे च सासहिम् ।  
+इन्द्रं देवासः शवसामदन्ननु ॥ १३ ॥'    
+
+इति त्रयोदशर्चमिन्द्रसूक्तम्, हिरण्यवर्णा इति नवर्चमप्सूक्तम् ॥ 
+
+[[708]]
+
+अथ निर्वापे विशेषः- आदित्यादिग्रहान्ते - “देवस्य त्वा॰ निर्ऋतये जुष्टं निर्वपामि, इन्द्राय॰, अद्भ्यः॰" इति। प्रोक्षणेऽप्येवमेव । आज्यनिर्वापाद्यग्निमुखान्तं कृत्वा 'आदित्याय त्वा' इत्यादि उदगपवर्गं यथाक्रमं चरुमभिमृश्याऽवदानधर्मेण ग्रहपक्वहोमान्ते “मूलं प्रजा॰, अहर्नो अद्य॰" इति द्वाभ्यां निर्ऋतये, “इन्द्रो ज्येष्ठां॰, पुरन्दरायः" इति द्वाभ्यामिन्द्राय, “ या दिव्या आपः॰, याश्च कूप्याः" इति द्वाभ्याम् अद्भ्यः क्रमेण पक्वं जुहुयात् ॥ ततस्स्विष्टकृतमवदायाऽन्तःपरिधौ निधाय - "मूलं प्रजां॰ अहनों अद्य॰ इन्द्रो ज्येष्ठां॰ पुरन्दराय या दिव्या आपः॰ याश्च कूप्याः॰ इति प्रतिमन्त्रमन्वाधानोक्तक्रमेण हुत्वाऽथाज्याहुतीरुपजुहोति - ॐ प्रजापतये स्वाहा ॥ मूलाय॰ ॥ प्रजायै॰ ॥ इन्द्राय॰ ॥ ज्येष्ठायै॰ ॥ ज्यैष्ठ्याय स्वाहा ॥ अभिजित्यैः ॥ अद्भ्यः॰ ॥ अषाढाभ्यः॰ ॥ समुद्राय॰ ॥ कामाय॰ ॥ अभिजित्यै॰ ॥ (यथालिङ्गं त्यागः) [चक्षुषोहे ते॰ ॥ १ ॥ यो मा चक्षुषा॰ ॥ २ ॥ यत्किञ्चासौ॰ ॥ ३ ॥ यातुधाना॰॥ ४ ॥ हन्मि तेऽहं॰ ॥ ५ ॥ अपनह्यामि॰ ॥ ६ ॥ पुरामुष्यः॥ ७ ॥ अन्ति दूरे॰ ॥ ८ ॥ यो मा नक्तम् ॥ ९ ॥ (सर्वत्र, निर्ऋतय इदं॰) तिष्ठा हरी रथ॰ ॥ १ ॥ कस्य वृषा॰ ॥ २ ॥ इन्द्रं वयं महाधने॰ ॥ ३ ॥ द्वितीयो वृत॰ ॥४॥ ससूर आ जनयन् ॥ ५ ॥  उत त्यदाश्व॰॥६॥ भरेष्विन्द्र॰॥७॥ महि क्षेत्रं॰ ॥ ८ ॥ उरुन्नो लोकं॰ ॥ ९ ॥ आ नो विश्वाभिः॰ ॥ १० ॥ इन्द्राय गावः ॥ ११ ॥ तास्ते वज्रिन्॰ ॥ १२ ॥ इमान्ते धियं॰ ॥ १३ ॥ (सर्वत्र, इन्द्रायेदं॰) हिरण्यवर्णा॰ ॥ १ ॥ यासाꣳ राजा॰ ॥ २ ॥ यासां देवाः॰ ॥ ३ ॥ शिवेन मा चक्षुषाः॰ ॥ ४ ॥ आपो भद्राः॰ ॥ ५ ॥ आदित्पश्यामि॰ ॥ ६ ॥ आपो हिष्ठा॰॥ ७ ॥ यो वः शिवतमो॰ ॥ ८ ॥ तस्मा अरइन्ग॰ ॥ ९ ॥ (सर्वत्र, अद्भ्य इदं॰) विश्वेभ्यो देवेभ्यः स्वाहा ॥ विष्णवे॰ ॥ वसुभ्यः॰ ॥ वरुणाय॰ ॥ अजायकपदे॰॥ अहये बुध्नियाय॰॥ पूष्णे॰॥ अश्विभ्यां॰॥ यमाय॰ ॥ अग्नये॰ ॥ प्रजापतये॰ ॥ सोमाय॰ ॥ रुद्राय॰ ॥ अदित्यै॰ ॥ बृहस्पतये॰ ॥ सर्पेभ्य॰ ॥ पितृभ्यः॰ ॥ अर्यम्णे॰ ॥ भगाय॰ ॥ सवित्रे॰ ॥ स्वष्ट्रे ॰ ॥ वायवे॰ ॥ इन्द्राग्निभ्यां॰ ॥ मित्राय॰ ॥ (यथालिङ्गं त्यागः) इति] ततः स्विष्टकृदादि प्रागुत्तरपरिषेकात्कृत्वा आदित्यादिनवग्रहदेवताभ्यः, इन्द्रादिलोकपालेभ्यः, निर्ऋतये, इन्द्राय, अद्भ्यः तत्तन्मन्त्रेण माषभक्तबलीन्दत्वा अन्ते क्षेत्रपालाय बलिं दद्यात् ॥
```

Comment:
[चक्षुषोहे ते॰ ॥ १ ॥ यो मा चक्षुषा॰ ॥ २ ॥ यत्किञ्चासौ॰ ॥ ३ ॥ यातुधाना॰॥ ४ ॥ हन्मि तेऽहं॰ ॥ ५ ॥ अपनह्यामि॰ ॥ ६ ॥ पुरामुष्यः॥ ७ ॥ अन्ति दूरे॰ ॥ ८ ॥ यो मा नक्तम् ॥ ९ ॥ (सर्वत्र, निर्ऋतय इदं॰) तिष्ठा हरी रथ॰ ॥ १ ॥ कस्य वृषा॰ ॥ २ ॥ इन्द्रं वयं महाधने॰ ॥ ३ ॥ द्वितीयो वृत॰ ॥४॥ ससूर आ जनयन् ॥ ५ ॥  उत त्यदाश्व॰॥६॥ भरेष्विन्द्र॰॥७॥ महि क्षेत्रं॰ ॥ ८ ॥ उरुन्नो लोकं॰ ॥ ९ ॥ आ नो विश्वाभिः॰ ॥ १० ॥ इन्द्राय गावः ॥ ११ ॥ तास्ते वज्रिन्॰ ॥ १२ ॥ इमान्ते धियं॰ ॥ १३ ॥ (सर्वत्र, इन्द्रायेदं॰) हिरण्यवर्णा॰ ॥ १ ॥ यासाꣳ राजा॰ ॥ २ ॥ यासां देवाः॰ ॥ ३ ॥ शिवेन मा चक्षुषाः॰ ॥ ४ ॥ आपो भद्राः॰ ॥ ५ ॥ आदित्पश्यामि॰ ॥ ६ ॥ आपो हिष्ठा॰॥ ७ ॥ यो वः शिवतमो॰ ॥ ८ ॥ तस्मा अरइन्ग॰ ॥ ९ ॥ (सर्वत्र, अद्भ्य इदं॰) विश्वेभ्यो देवेभ्यः स्वाहा ॥ विष्णवे॰ ॥ वसुभ्यः॰ ॥ वरुणाय॰ ॥ अजायकपदे॰॥ अहये बुध्नियाय॰॥ पूष्णे॰॥ अश्विभ्यां॰॥ यमाय॰ ॥ अग्नये॰ ॥ प्रजापतये॰ ॥ सोमाय॰ ॥ रुद्राय॰ ॥ अदित्यै॰ ॥ बृहस्पतये॰ ॥ सर्पेभ्य॰ ॥ पितृभ्यः॰ ॥ अर्यम्णे॰ ॥ भगाय॰ ॥ सवित्रे॰ ॥ स्वष्ट्रे ॰ ॥ वायवे॰ ॥ इन्द्राग्निभ्यां॰ ॥ मित्राय॰ ॥ (यथालिङ्गं त्यागः) इति]

ityakhilaṁ []-āvaraṇēna saha bhinna pāṭhakhaṇḍē (paragraph) syāt. tasya coddharaNacihnaM syāt. tathā, bhinna-mantrapratīkāḥ bhinna-paṅktiṣu.

Reply:
यथा भवता उक्तं मया कृतम् - भिन्न पाठकाण्डं, उद्धारणचिह्नम्, भिन्नमन्त्रप्रतीकाः भिन्नपङ्तिषु (The last one, to the extent I could understand)

Reply:
एवम् इष्यते - 

```
[

> 'चक्षुषोहे ते॰ ॥ १ ॥   
यो मा चक्षुषा॰ ॥ २ ॥   
यत्किञ्चासौ॰ ॥ ३ ॥   
यातुधाना॰॥ ४ ॥   
हन्मि तेऽहं॰ ॥ ५ ॥   
अपनह्यामि॰ ॥ ६ ॥   
पुरामुष्यः॥ ७ ॥   
अन्ति दूरे॰ ॥ ८ ॥   
यो मा नक्तम् ॥ ९ ॥   
(सर्वत्र, निर्ऋतय इदं॰)'

>  तिष्ठा हरी रथ॰ ॥ १ ॥   
कस्य वृषा॰ ॥ २ ॥   
इन्द्रं वयं महाधने॰ ॥ ३ ॥   
द्वितीयो वृत॰ ॥४॥ 
...
```

Reply:
एतावत् मया अधुना शोषितम्